tests/factories/pages.py

```diff
@@ -74,3 +77,35 @@ class WagtailPageFactory(factory.DjangoModelFactory):
 
     class Meta:
         model = Page
+
+    @classmethod
+    def _create(cls, *args, **kwargs):
```

Comment:
Don't think we need this since we're mostly using a pytest fixture when creating a site tree. Or is there any particular reason why you implemented this?